Localization/en-US/Mods.PathOfTerraria.SkillPassives.hjson

```diff
@@ -0,0 +1,24 @@
+# Passives will always be in the format
+# Name:
+# Tooltip:
+# You do not need to generate these yourself, run the mod and it'll add the keys automatically.
+
+Anchor: {
+	Name: Anchor
+	Tooltip: Your journey starts here
+}
+
+"Lightning Nova": {
```

Comment:
Did you manually write these? You don't need "" around keys in hjson and if they're manually written we should go ahead and automatically generate them instead, especially for future skill passives.

Reply:
When you have a space, the " is required or you get a parse error. I did manually write these as a temp thing. I was noticing that `GetType().Name` had the space and this is why it was required.

Reply:
You're right, not used to names in keys. GetType().Name shouldn't have a space though, that isn't syntactically possible - it's just the class name as-is. Spacing in the key will cause issues for getting stuff in the future, and is just awkward generally.

Reply:
Thats what I thought as well. https://github.com/Path-of-Terraria/PathOfTerraria/pull/345/files#diff-40ab9976403f9797d324ce98efc584d56dc0c1e73e686fa0dbb9ea68e4dfd723R19
But this had a space for whatever weird reason.

Reply:
Nvm I'm stupid. https://github.com/Path-of-Terraria/PathOfTerraria/pull/345/files#diff-bdb241860420ee2afca4dddc9f01ed32c3bf52df1c9156b2d387b11cdf03aeefR12
I was overriding it for whatever reason.
Although I swear I looked at that lol

Reply:
I think the a lot of systems in the mod used Name as the display name directly, so chances are you either autopiloted or referenced another thing that did so. No worries! lol

Reply:
Fixed